app/src/main/java/com/mapzen/sample/pelias/MainActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.mapzen.sample.pelias;
+
+import com.mapzen.pelias.BoundingBox;
+import com.mapzen.pelias.Pelias;
+import com.mapzen.pelias.PeliasLocationProvider;
+import com.mapzen.pelias.gson.Result;
+import com.mapzen.pelias.widget.AutoCompleteAdapter;
+import com.mapzen.pelias.widget.AutoCompleteListView;
+import com.mapzen.pelias.widget.PeliasSearchView;
+
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class MainActivity extends AppCompatActivity {
+
+  private static final String TAG = MainActivity.class.getSimpleName();
+
+  AutoCompleteListView listView;
+  Pelias pelias;
+
+  //Hard code to NYC for simplicity
+  PeliasLocationProvider peliasLocationProvider = new PeliasLocationProvider() {
+    @Override public double getLat() {
+      return 40.74433;
+    }
+
+    @Override public double getLon() {
+      return -73.9903;
+    }
+
+    @Override public BoundingBox getBoundingBox() {
+      return new BoundingBox(40.74433, -73.9903, 40.741050, -73.996142);
+    }
+  };
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+    setupPelias();
+    setupListView();
+    setupPeliasSearchView();
+  }
+
+  private void setupPelias() {
+    pelias = new Pelias();
+    pelias.setLocationProvider(peliasLocationProvider);
+  }
+
+  private void setupListView() {
+    listView = (AutoCompleteListView) findViewById(R.id.list_view);
+    AutoCompleteAdapter autocompleteAdapter = new AutoCompleteAdapter(this, android.R.layout.simple_list_item_1);
```

Comment:
If `checkstyle` is running properly for the sample app code it _should_ complain about this line over 100 characters